package/etc/context_templates/splunk_metadata.csv.example

```diff
@@ -120,3 +120,7 @@ zscaler_lss,index,netproxy
 unknown,index,main
 unknown,source,SC4S:unknown
 unknown,sourcetype,SC4S:unknown
+Cisco_C100V Email Security Virtual Appliance_ESA_CONSOLIDATED_LOG_EVENT,sc4s_template,t_cef_hdr_msg_for_esa
```

Comment:
It Seems CISCO is unsurprising and incorrectly including the hardware model in the CEF header that means we need to get from CISCO each of the product strings they use for the physical and cloud versions and list them each
also change the template to t_legacy_hdr_msg

Reply:
Are we going to revert the changes in **t_legacy_hdr_msg** template?